CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ endif()
 # the libarchive dependencies
 if(LibArchive_LIBRARIES AND LibArchive_INCLUDE_DIRS)
   set(LibArchive_FOUND ON)
+  message("Using flags for LibArchive ${LibArchive_LIBRARIES} and ${LibArchive_INCLUDE_DIRS}")
```

Comment:
Please keep cmake silent.     If you want to know use `grep LibArchive CMakeCache,txt`

Reply:
The only case where this message would happen is if the programmer adds the flags for LibArchive_LIBRRIES and LibArchive_INCLUDE_DIRS. I found it useful to confirm that the overrides had taken effect.

But if you think it should be silent even if they're overrides, I'll remove the message.

Reply:
Yes please, I'd like to keep cmake pretty silent.   I know it is only one message, but there is a lot to configure and overrule ...   And yes, I also frequently (temporary) add message() calls to CMake files.